example-conf.yml

```diff
@@ -566,6 +569,25 @@ relay:
     #
     #clientbot_modesync: none
 
+###
+#
+# Serv Protect
+#
+# If you use this please uncomment the following values
+# and set them to what you want them to be.
+#
+###
```

Comment:
Remove the supersized block headers please, there's none of that anywhere else in the example conf.